src/pages/build-apps/guides/transaction-signing.md

```diff
@@ -264,4 +326,4 @@ const MyComponent = () => {
 
 ## Request testnet STX from faucet
 
-You may find it useful to request testnet STX from [the faucet](https://testnet.blockstack.org/faucet) while developing your app with the Stacks testnet.
+You may find it useful to request testnet STX from [the Stacks Explorer sandbox](https://explorer.stacks.co/sandbox?chain=testnet) while developing your app with the Stacks testnet.
```

Comment:
should we attach the `?chain=` param in links to not?

Reply:
I think it makes sense for testnet. I know the sandbox is testnet only at this time, but if we have a mainnet sandbox (without a faucet obviously) then this would probably be the best URL.